provision/aws/rds/aurora_create_peering_connection.sh

```diff
@@ -63,10 +65,24 @@ ROSA_PUBLIC_ROUTE_TABLE_ID=$(aws ec2 describe-route-tables \
   --query "RouteTables[*].RouteTableId" \
   --output text
 )
-aws ec2 create-route \
-  --route-table-id ${ROSA_PUBLIC_ROUTE_TABLE_ID} \
-  --destination-cidr-block ${AURORA_VPC_CIDR} \
-  --vpc-peering-connection-id ${PEERING_CONNECTION_ID}
+
+
+EXISTS=$(aws ec2 describe-route-tables\
+  --filters "Name=route-table-id,Values=${ROSA_PUBLIC_ROUTE_TABLE_ID}" \
+    "Name=route.destination-cidr-block,Values=${AURORA_VPC_CIDR}" \
+    "Name=route.vpc-peering-connection-id,Values=${PEERING_CONNECTION_ID}" \
+  --query "RouteTables[*].RouteTableId" \
+  --output text)
+
+if [ "${EXISTS}" == "" ]; then
+  aws ec2 delete-route \
```

Comment:
Why do we need to delete and then create if we've already checked that the route doesn't exist?

Reply:
I had some runs where `EXISTS` was empty but the route really existed. I'll remove it since I'm pretty sure I was doing something wrong back then :rofl: